src/transactionPayload.ts

```diff
@@ -66,22 +66,22 @@ export class TransactionPayload {
     }
 
     /**
-     * Returns a new builder, to be used for contract deploy transactions.
-     */
+    * @deprecated Use {@link SmartContract} to create transactions related to smart contracts.
```

Comment:
to create smart contracts related transactions

also below

Reply:
Fixed.